app/main.py

```diff
@@ -1,25 +1,96 @@
+from __future__ import annotations
 from abc import ABC
+from typing import Any, Callable
```

Comment:
In latest versions of python you can use this without imports, simply write in lowercase `Any` -> `any`